### PR TITLE
Scopes

### PR DESCRIPTION
**Pull request by mrate**
Fix for #1360

### CODE REVIEW

File: src/org/opensolaris/opengrok/analysis/JFlexXref.java [hunk 1]
```diff
@@ -308,7 +308,7 @@ public void write(Writer out) throws IOException {
 
         // terminate scopes
```
Comment (tulinkry) [inline src/org/opensolaris/opengrok/analysis/JFlexXref.java]: copyright 2017

**Comment by vladak** [comment]
So, after this change are all the tags in the page closed ?

**Comment by vladak** [comment]
Also, some sort of unit test would be nice.

**Comment by vladak** [comment]
Verified that the change fixes the footer problem and HTML code comes out clean (at least FF does not mark any tags red anymore)

**Pull request merged**